src/services/StreamLabs.ts

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+
+interface StreamLabsConfig {
+	token: string;
+}
+
+interface StreamLabsLogger {
+	log: (levels: string[], log: any) => void;
```

Comment:
This is actually 

```
log: (tags: [], log: string|object) => void
```

We'll update it when we decide how to improve the logging system (see #103)